tests/framework/helpers/HtmlActiveTest.php

```
@@ -0,0 +1,470 @@
+    {
+        Yii::$app->request->enableCsrfValidation = true;
+        Yii::$app->request->cookieValidationKey = 'foobar';
+
+        $token = Yii::$app->request->getCsrfToken();
+
+        self::assertSame(
+            <<<HTML
+            <form id="mycsrfform" action="/index.php" method="post">
+            <input type="hidden" name="_csrf" value="{$token}">
+            HTML,
+            Html::beginForm('/index.php', 'post', ['id' => 'mycsrfform']),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\helpers\Html' in method 'testCsrfDisable'. (undefined)

(StaticAccess)